pkg/conf/mesos_target_conf.go

```diff
@@ -0,0 +1,225 @@
+package conf
+
+import (
+	"io/ioutil"
+	"encoding/json"
+	"github.com/golang/glog"
+	"fmt"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+	"github.com/turbonomic/turbo-go-sdk/pkg/probe"
+)
+
+// Configuration Parameters to connect to a Mesos Target
+type MesosTargetConf struct {
```

Comment:
Maybe do a nested json, like what we did in [TurboCommunicationConfig](https://github.com/turbonomic/turbo-go-sdk/blob/master/pkg/service/turbo_communication_config.go#L28) in go-sdk.